e2e-tests/tests/add-page-with-layout.spec.ts

```diff
@@ -9,12 +9,15 @@ const expectedPage = fs.readFileSync(
 
 studioTest("can add a page using a layout", async ({ page, studioPage }) => {
   const pageName = "LayoutPage";
-  await studioPage.takePageScreenshotAfterImgRender();
   const pageInTree = page.getByText(pageName);
   await expect(pageInTree).toHaveCount(0);
-  await studioPage.addStaticPage(pageName, "index.html", "LocationLayout");
+
+  const addPageSection = studioPage.addPageSection;
+  await addPageSection.selectPageType(false);
+  await addPageSection.enterBasicPageData(pageName, "index.html");
+  await addPageSection.selectLayout("LocationLayout", true);
+
   await expect(pageInTree).toHaveCount(1);
-  await studioPage.takePageScreenshotAfterImgRender();
```

Comment:
I noticed pretty much all our tests have a screenshot from before we click the Save button and a screenshot for after. Are we planning on removing this "before" screenshot from all of the tests?

Reply:
Yep, that's the plan.